--- sosi2osm-20140707git634fc65.orig/tag.cpp
+++ sosi2osm-20140707git634fc65/tag.cpp
@@ -37,7 +37,7 @@ void loadLua(char* filename) {
     luaL_openlibs(state);
     
     char library_filename[256];
-    snprintf(library_filename, 256, "%s/lua/sosi2osm.lua", dirname(execname));
+    snprintf(library_filename, 256, "/usr/share/sosi2osm/lua/sosi2osm.lua");
     
     int r = luaL_loadfile(state, library_filename);
     if (r) {
@@ -46,10 +46,23 @@ void loadLua(char* filename) {
     }
     lua_call(state, 0, 0);
     
-    r = luaL_loadfile(state, filename);
-    if (r) {
-        fprintf(stderr, "Failed to load lua file %s\n", filename);
-        exit(1);
+    FILE *file;
+    if((file = fopen(filename, "r")))
+    {
+        fclose(file);
+        r = luaL_loadfile(state, filename);
+        if(r) {
+            fprintf(stderr, "Failed to load lua file %s\n", filename);
+            exit(1);
+        }
+    }
+    else {
+        snprintf(library_filename, 256, "/usr/share/sosi2osm/lua/%s",filename);
+        r = luaL_loadfile(state, library_filename);
+        if(r) {
+            fprintf(stderr, "Failed to load lua file %s\n", filename);
+            exit(1);
+        }
     }
     
     
